node-js-basics-101/node_modules/asynquence-contrib/contrib.js

```diff
@@ -0,0 +1,5 @@
nts);return a.call(t,r.length>1?r:r[0]),e.messages.apply(t,r)}).or(function $$or(){var e=n.call(arguments);u.call(t,e.length>1?e:e[0])})})}}),e.extend("try",function $$extend(r,a){return function $$try(){if(a("seq_error")||a("seq_aborted")||0===arguments.length)return r;var u=n.call(arguments).map(function $$map(r){return function $$then(a){var u=n.call(arguments),l=e.apply(t,u.slice(1));l.then(function $$inner$then(){r.apply(t,arguments)}).val(function $$val(){a.apply(t,arguments)}).or(function $$inner$or(){var n=e.messages.apply(t,arguments);a({"catch":n.length>1?n:n[0]})})}});return r.then.apply(t,u),r}}),e.extend("until",function $$extend(r,a){return function $$until(){if(a("seq_error")||a("seq_aborted")||0===arguments.length)return r;var u=n.call(arguments).map(function $$map(r){return function $$then(a){var u=n.call(arguments),l=e.apply(t,u.slice(1));l.then(function $$inner$then(){var e=n.call(arguments);e[0]["break"]=function $$break(){a.fail.apply(t,arguments),l.abort()},r.apply(t,e)}).val(function $$val(){a.apply(t,arguments)}).or(function $$inner$or(){$$then.apply(t,u)})}});return r.then.apply(t,u),r}}),e.extend("waterfall",function $$extend(r,a){return function $$waterfall(){if(a("seq_error")||a("seq_aborted")||0===arguments.length)return r;var u=n.call(arguments);return r.then(function $$then(r){var a=e.messages(),l=e.apply(t,n.call(arguments,1));u.forEach(function $$each(n){l.then(n).val(function $$val(){var n=e.messages.apply(t,arguments);return a.push(n.length>1?n:n[0]),a})}),l.pipe(r)}),r}}),e.wrap=function $$wrap(r,a){function checkThis(e,n){return!e||"undefined"!=typeof window&&e===window||"undefined"!=typeof global&&e===global?n:e}function paramSpread(e){return regeneratorRuntime.mark(function paramSpread(n){return regeneratorRuntime.wrap(function paramSpread$(t){for(;;)switch(t.prev=t.next){case 0:return t.delegateYield(e.apply(this,n.messages),"t0",1);case 1:case"end":return t.stop()}},paramSpread,this)})}var u,l,c,s,o;if(a=a&&"object"==typeof a?a:{},a.errfcb&&a.splitcb||a.errfcb&&a.simplecb||a.splitcb&&a.simplecb||"errfcb"in a&&!a.errfcb&&!a.splitcb&&!a.simplecb||a.params_first&&a.params_last||a.spread&&!a.gen)throw Error("Invalid options");return s=a["this"]&&"object"==typeof a["this"]?a["this"]:t,u=a.errfcb||!(a.splitcb||a.simplecb),l=!!a.params_first||!a.params_last&&!("params_first"in a||a.params_first)||"params_last"in a&&!a.params_first&&!a.params_last,o=!(!a.spread&&"spread"in a),c=l?"push":"unshift",a.gen?(o&&(r=paramSpread(r)),function $$wrapped$gen(){return e(e.messages.apply(t,arguments)).runner(r)}):u?function $$wrapped$errfcb(){var t=n.call(arguments),a=checkThis(this,s);return e(function $$asq(e){t[c](e.errfcb),r.apply(a,t)})}:a.splitcb?function $$wrapped$splitcb(){var t=n.call(arguments),a=checkThis(this,s);return e(function $$asq(e){t[c](e,e.fail),r.apply(a,t)})}:a.simplecb?function $$wrapped$simplecb(){var t=n.call(arguments),a=checkThis(this,s);return e(function $$asq(e){t[c](e),r.apply(a,t)})}:void 0},e});
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.
* 's' is already defined.
* Functions declared within loops referencing an outer scoped variable may lead to confusing semantics.
* Expected a conditional expression and instead saw an assignment.
* Expected a 'break' statement before 'case'.
* Too many errors. (83% scanned).

<!-- Comment by [SideCI](https://sideci.com) -->